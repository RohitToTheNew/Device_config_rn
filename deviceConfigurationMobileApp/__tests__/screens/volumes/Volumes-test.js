import {NavigationContainer} from '@react-navigation/native';
import React from 'react';
import renderer from 'react-test-renderer';
import Volumes from '../../../src/screens/volumes';

describe('on successful auth with BLE', () => {
  let tree;

  beforeEach(() => {
    tree = renderer
      .create(
        <NavigationContainer>
          <Volumes />
        </NavigationContainer>,
      )
      .toJSON();
  });

  it('should renders volumes screen correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
