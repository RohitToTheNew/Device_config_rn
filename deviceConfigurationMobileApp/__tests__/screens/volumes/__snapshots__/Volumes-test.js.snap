// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`on successful auth with BLE should renders volumes screen correctly 1`] = `null`;
